django/db/backends/base/operations.py

```diff
@@ -433,7 +433,26 @@ def validate_autopk_value(self, value):
         """
         return value
 
-    def value_to_db_date(self, value):
+    def adapt_unknown_value(self, value):
+        """
+        Transforms a value to something compatible with the backend driver.
+
+        This method only depends on the type of the value. It's designed for
+        cases where the target type isn't known. As a consequence it may not
```

Comment:
It might be helpful for clarity to give an example of such a case here. (As far as I'm aware the only actual case is raw queries, correct?)